packages/manager/apps/pci-private-network/src/types/private-network-form.type.ts

```diff
@@ -0,0 +1,22 @@
+export type Subnet = {
+  cidr: string;
+  enableDhcp: boolean;
+  enableGatewayIp: boolean;
+  ipVersion: number;
+};
+
+export type Gateway = {
+  model: string;
+  name: string;
+};
+
+export type NewPrivateNetworkForm = {
+  region: string;
+  isLocalZone: boolean;
+  name: string;
+  defaultVlanId: number;
+  vlanId: number;
+  subnet: Subnet;
+  gateway?: Gateway;
+  existingGatewayId?: string | boolean;
+};
```

Comment:
It's optionnal but when you are the first use of Zod on this project. I think is good have an example of use.

I have for suggestion split your Zod schema validator and use zod infer to generate your type. It's more maintainable

```ts
export type NewPrivateNetworkForm = z.infer<typeof NEW_PRIVATE_NETWORK_FORM_SCHEMA >
```